cmd/collector/app/handler/testdata/zipkin_thrift_v1_merged_spans.json

```diff
@@ -0,0 +1,79 @@
+[
```

Comment:
I used the existing `zipkin_v1_merged_spans.json` file (Zipkin JSON v1), converged into object model, and re-serialized as JSON. It uses different representations for many fields, like raw numbers instead of hex strings for IDs.